kbase-extension/static/kbase/css/narrative.css

```diff
@@ -23,56 +19,40 @@ body {
 }
 
 html {
-  background-color: #a2a2a2;
-
-  /* border: 1px solid #000; */
   padding: 0;
```

Comment:
I believe all the rest of these are default values (or otherwise unspecified for the root `<html>` element), so maybe this whole rule set is redundant?

Reply:
I am sure there are probably other similarly redundant rulesets. I've deleted this one now.